tsfc/driver.py

```diff
@@ -282,6 +282,7 @@ def compile_expression_dual_evaluation(expression, to_element, coordinates, *,
     :arg parameters: parameters object
     :arg coffee: compile coffee kernel instead of loopy kernel
     """
+
```

Comment:
Unnecessary change.

Reply:
Not sure how it happened.  Putting it back.